src/main/golo/ansicodes.golo

```diff
@@ -0,0 +1,64 @@
+# ............................................................................................... #
+#
+# Copyright (c) 2012-2015 Institut National des Sciences Appliquées de Lyon (INSA-Lyon)
+#
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# ............................................................................................... #
+
+----
+This module provides helper functions to deal with ANSI escape codes for console outputs.
+
+All function names are self-explanatory. Note that `ansi_likely_supported` tests if the current
+operating system is _likely_ to support ANSI codes. It merely checks that the host operating
+system is not MS Windows.
+
+See also: [Wikipedia on ANSI escape codes](https://en.wikipedia.org/wiki/ANSI_escape_code)
+----
+module gololang.AnsiCodes
+
+function ansi_likely_supported = ->
+  not System.getProperty("os.name"): contains("Windows")
```

Comment:
why so much hate for windows
https://github.com/adoxa/ansicon
personally I use it with cygwin, it works like a charm